exec/java-exec/src/main/java/org/apache/drill/exec/planner/logical/DrillJoinRel.java

```diff
@@ -45,6 +46,7 @@
  */
 public class DrillJoinRel extends DrillJoinRelBase implements DrillRel {
   public static final String EQUALITY_CONDITION = "==";
+  boolean isSemiJoin;
```

Comment:
Since we already have DrillSemiJoinRel as a logical join,  it is better to not add this flag here.

Reply:
@amansinha100 done. Please take a look.